backend/clickhouse/query.go

```diff
@@ -495,3 +498,65 @@ func KeyValuesAggregated(ctx context.Context, client *Client, tableName string,
 	span.Finish(rows.Err())
 	return values, rows.Err()
 }
+
+func matchesQuery[TObj interface{}, TReservedKey ~string](row *TObj, config tableConfig[TReservedKey], filters *queryparser.Filters) bool {
+	v := reflect.ValueOf(*row)
+
+	rowBodyTerms := map[string]bool{}
+	if config.bodyColumn != "" && len(filters.Body) > 0 {
+		for _, field := range strings.Fields(v.FieldByName(config.bodyColumn).String()) {
```

Comment:
testing for case sensitivity, handling non-alpha + non-whitespace characters (like `, . ; :`)